.zenodo.json

```diff
@@ -78,7 +78,97 @@
       "orcid": "0000-0001-8421-365X"
     },
     {
-     "affiliation": "Université Paris-Saclay, Inria, CEA, Palaiseau, France",
+      "affiliation": "",
+      "name": "Ariel Rokem ",
+      "orcid": "0000-0003-0679-1985"
+     },
+     {
+      "affiliation": "",
+      "name": "Austin Hurst",
+      "orcid": "0000-0002-0219-4131"
+     },
+     {
+      "affiliation": "",
+      "name": "Diego Lozano-Soldevilla",
+      "orcid": "0000-0003-1794-8204"
+     },
+     {
+      "affiliation": "",
+      "name": "Eduard Ort",
+      "orcid": "0000-0001-5546-3561"
+     },
+     {
+      "affiliation": "",
+      "name": "Ethan Knights",
+      "orcid": "0000-0001-6078-9160"
+     },
+     {
+      "affiliation": "",
+      "name": "Evgenii Kalenkovich",
+      "orcid": "0000-0002-4606-4179"
+     },
+     {
+      "affiliation": "",
+      "name": "Franziska von Albedyll",
+      "orcid": ""
+     },
+     {
+      "affiliation": "",
+      "name": "Fu-Te Wong",
+      "orcid": ""
+     },
+     {
+      "affiliation": "",
+      "name": "Jean-Rémi King",
+      "orcid": "0000-0002-2121-170X"
+     },
+     {
+      "affiliation": "",
+      "name": "Julia Guiomar Niso Galán",
+      "orcid": "0000-0001-5872-8924"
+     },
+     {
+      "affiliation": "",
+      "name": "Richard Koehler",
+      "orcid": "0000-0002-5219-1289"
+     },
+     {
+      "affiliation": "",
+      "name": "Robert Luke",
+      "orcid": "0000-0002-4930-8351"
+     },
+     {
+      "affiliation": "",
+      "name": "Simon Kern",
+      "orcid": ""
+     },
+     {
+      "affiliation": "",
+      "name": "Sin Kim",
+      "orcid": ""
+     },
+     {
+      "affiliation": "",
+      "name": "Sophie Herbst",
+      "orcid": "0000-0001-8502-0366"
+     },
+     {
+      "affiliation": "",
+      "name": "Swastika Gupta",
+      "orcid": ""
+     },
+     {
+      "affiliation": "",
+      "name": "Tom Donoghue",
+      "orcid": "0000-0001-5911-0472"
```

Comment:
@TomDonoghue please check and/or comment with missing details!

Reply:
The ORCiD is correct! To match my ORCiD full / real name I guess it should be "Thomas Donoghue". 
If you want to add a current affiliation it's Columbia University (department of biomedical engineering)